docs/docs/components.mdx

```diff
@@ -824,11 +824,16 @@ Note: The `feature-dimension` for sequence and sentence features does not have t
   Make the featurizer case insensitive by adding the `case_sensitive: False` option, the default being
   `case_sensitive: True`.
 
+  To correctly process languages such as Chinese that don't use whitespace for word separation,
+  the user needs to add the `use_word_boundaries: False` option, the default being `use_word_boundaries: True`.
+
   ```yaml-rasa
   pipeline:
   - name: "RegexFeaturizer"
     # Text will be processed with case sensitive as default
     "case_sensitive": True
+    # use match word boundaries for lookup table
```

Comment:
```suggestion
    # use match word boundaries for regexes and lookup table
```

Reply:
If the `regexes` in here means `training_data.regex_features`, as far as I know, `RegexFeaturizer` and `RegexEntityExtractor` use `training_data.regex_features` without modification, if so, word boundaries option will not affect the `regexes`.